src/components/Builder/Description/Description.scss

```diff
@@ -0,0 +1,8 @@
+.description {
+    max-width: 60vw;
```

Comment:
Do we need a max width? I think the parent should handle widths, then this component can be more agnostic and be reused in other places. This seem like a too specific to the page styling.